### ISSUE TITLE
Installation from homebrew

### ISSUE THREAD

**Issue by jwoertink** [jwoertink/fez#14]
https://github.com/jwoertink/homebrew-fez/issues/2

I'm not sure why this doesn't work.

**Comment by jwoertink** [comment]
Looks like https://github.com/jwoertink/homebrew-fez/commit/eec4aa0bcc45b3e5c94e621b1d0328a7ba0dea0d fixes it. Just having it download the binary.

**Comment by divinerites** [comment]
Hello,
It seems that the binary is not available from the latest 0.10.0 (only for 0.9.1)

```
$ brew install fez
==> Installing fez from jwoertink/fez
==> Downloading https://github.com/jwoertink/fez/releases/download/v0.10.0/fez
curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "fez"
Download failed: https://github.com/jwoertink/fez/releases/download/v0.10.0/fez
```
